### ISSUE TITLE
Problem with --pyz if modules require other modules to run

### ISSUE THREAD

**Issue by nextized** [liftoff/pyminifier#31]
I like pyminifier and use it a lot. 

There's a minor issue if a module imported into __init.py__ has some other imports, which are required for the imported modules but not for __init__.py. Those are not included. So the finished file does not work (module xy not found).

Thanks you for fixing this issue soon.

**Comment by liftoff** [comment]
I assume you have this problem when using the --pyz option?  Because if you minify your dependencies individually it should still work fine.

I'll check the code that does the --pyz thing to make sure it looks at imports inside of `__init__.py` files anyway...

**Comment by hdittakavi** [comment]
Thank you for all the effort you are putting into this. Is this issue --pyz not including the dependencies resolved yet? If not when will this be resolved?

**Comment by liftoff** [comment]
Yeah, I still haven't addressed this one yet.  It's still in the TODO.
